package-lock.json

```diff
@@ -1,8 +1,382 @@
 {
   "name": "@crowdin/cli",
   "version": "3.9.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
This file shouldn't be updated

Reply:
There are still some changes to this file. In my opinion, it shouldn't be changed in this PR at all